package.json

```diff
@@ -139,10 +140,10 @@
     "react-split-pane": "^0.1.57",
     "react-toolbox": "^1.2.1",
     "react-topbar-progress-indicator": "^1.0.0",
-    "react-transition-group": "1.x",
-    "react-waypoint": "^3.1.3",
+    "react-transition-group": "2.2.1",
```

Comment:
Use a range here (or mention in a GH comment why we shouldn't).

Reply:
Oops, missed that.